www/doc/1.7/report.html

```diff
@@ -20,143 +22,176 @@ <h1>Herbie reports</h1>
     </nav>
   </header>
 
-  
   <h1>The Herbie report</h1>
 
   <p>When used <a href="using-web.html">in the browser</a>, Herbie
-  generates HTML reports full of information about the accuracy of
-  your input and its output expressions.</p>
+    generates HTML reports full of information about the accuracy of
+    your input and its output expressions.</p>
 
   <h2 id="summary">Summary numbers</h2>
 
   <p>First, a brief summary of the results. For most uses, the
-    “Average Error” number, which describes how accurate the input and
-    output expressions are, is the most important statistic in this
-    section. The other numbers list time Herbie took to improve the
-    program and the <a href="input.html#precisions">precision</a>
-    assumed for floating-point operations.
+    "Percentage Accurate" and "Speedup" metric, which describes the change in accuracy and speed up from
+    the starting expression to what Herbie thinks might be the best alterative,
+    is the most important statistics in this section. The other numbers list
+    time Herbie took to improve the program and number of other alternatives
+    found.
   </p>
 
   <figure>
     <img width="100%" src="report-large.png" />
-    <figcaption>Summary numbers from a Herbie report. The "binary64"
-    precision refers to double-precision IEEE-754
-    arithmetic.</figcaption>
+    <figcaption>Summary numbers from a Herbie report.</figcaption>
   </figure>
 
-  <h2 id="programs">Input and output programs</h2>
+  <ul>
+    <li><b>Percentage Accurate</b>: The change in accuracy from
+      the starting expression to what Herbie thinks might be the best alterative.</li>
```

Comment:
```suggestion
      the starting expression to what Herbie thinks might be the most accurate alternative.</li>
```

Reply:
Also instead of "starting expression" use "initial program" so that we always use the same terminology.